<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>391</width>
      <height>551</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#widget{
	
	background-image: url(:/res/background.png);
	border: 1px solid rgb(222, 215, 208);
	border-radius: 15px;
}
font: '微软雅黑';
</string>
    </property>
    <widget class="QToolButton" name="btnSet">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>10</y>
       <width>24</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;
background-color: rgba(0 , 0, 0, 0);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resourse.qrc">
       <normaloff>:/res/Search.png</normaloff>:/res/Search.png</iconset>
     </property>
    </widget>
    <widget class="QToolButton" name="btnClose">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>10</y>
       <width>24</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton{
	border: none;
	
	background-color: rgba(0 , 0, 0, 0);
}
QToolButton:hover{
	background-color: rgb(255, 0 , 0);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resourse.qrc">
       <normaloff>:/res/close.png</normaloff>:/res/close.png</iconset>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>191</width>
       <height>281</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255,0);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resourse.qrc">:/res/touxiang.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>260</y>
       <width>271</width>
       <height>271</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="verticalSpacing">
       <number>24</number>
      </property>
      <item row="0" column="0">
       <widget class="QLineEdit" name="editUser">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
                color: gray; /* 默认文本颜色为浅灰色 */
				border-radius: 15px;
				font: '微软雅黑';
                font-size: 14px;
            }

            QLineEdit:focus {
                color: black; /* 输入时文本颜色为黑色 */
				border-radius: 15px;
				font: '微软雅黑';
                font-size: 14px;
            }
</string>
        </property>
        <property name="text">
         <string>输入QQ号</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="editPassword">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
                color: gray; /* 默认文本颜色为浅灰色 */
				border-radius: 15px;
                font-size: 14px;
            }
QLineEdit:focus {
                color: black; /* 输入时文本颜色为黑色 */
				border-radius: 15px;
                font-size: 14px;
            }</string>
        </property>
        <property name="text">
         <string>输入QQ密码</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="btnArticle">
        <property name="styleSheet">
         <string notr="true"> color: gray; /* 默认文本颜色为浅灰色 */</string>
        </property>
        <property name="text">
         <string>已阅读并同意服务协议和QQ隐私保护指引</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="btnSignIn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">
background-color: rgb(170, 214, 255);
border-radius:10px;
color: rgb(255, 255, 255);

</string>
        </property>
        <property name="text">
         <string>登录</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="leftMargin">
         <number>50</number>
        </property>
        <property name="rightMargin">
         <number>50</number>
        </property>
        <item>
         <widget class="QPushButton" name="btnMore">
          <property name="styleSheet">
           <string notr="true">border:none;
color: rgb(45, 119, 229);
border-radius:10px;
</string>
          </property>
          <property name="text">
           <string>更多选项</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>|</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSignUp">
          <property name="styleSheet">
           <string notr="true">border:none;
color: rgb(45, 119, 229);
border-radius:10px;
</string>
          </property>
          <property name="text">
           <string>注册账号</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resourse.qrc"/>
 </resources>
 <connections/>
</ui>
